src/test/java/io/openliberty/tools/intellij/lsp4jakarta/it/cdi/ManagedBeanTest.java

```diff
@@ -104,46 +104,92 @@ public void scopeDeclaration() throws Exception {
         Diagnostic d1 = d(12, 16, 17,
                 "Scope type annotations must be specified by a producer field at most once.",
                 DiagnosticSeverity.Error, "jakarta-cdi", "InvalidScopeDecl");
-        d1.setData(new Gson().toJsonTree(Arrays.asList("Dependent", "ApplicationScoped", "Produces")));
+        d1.setData(new Gson().toJsonTree(Arrays.asList("jakarta.enterprise.context.Dependent",
+                "jakarta.enterprise.context.ApplicationScoped", "Produces")));
```

Comment:
This expected output is incorrect. All class names in the list should be fully qualified.

Reply:
Done